packages/beacon-state-transition/test/perf/altair/epoch/processInactivityUpdates.test.ts

```diff
@@ -21,6 +21,10 @@ import {mutateInactivityScores} from "./util";
 
 describe("altair processInactivityUpdates", () => {
   const vc = numValidators;
+  setBenchOpts({
+    maxMs: 60 * 1000,
+    minMs: 50 * 1000,
```

Comment:
Why are this parameters necessary?

Reply:
I read the benchmark result wrong, thought the test was stable.
Reverted the change.